### PR TITLE
Feature/36048 row action changes

### PR DESCRIPTION
**Pull request by ankitsaraogi**
Table row changes as per the tp ticket.
1. tested the tables with different row actions available.
2. Providing support for expression and render to show row action
3, Bug fixes for combobox.

### CODE REVIEW

File: src/components/ComboBox/ComboBox.tsx [hunk 1]
```diff
@@ -45,11 +46,21 @@ class ComboBox extends React.Component<Props, State> {
...
       // In case use is searching something, and then removes its search text, combobox shud list the initialItem
       // Therefore, keeping copy of it so that its not lose, as items is changed depending on search and selection.
-      initialItems: JSON.parse(JSON.stringify(props.items)),
+      initialItems: this.addRenderer(props.items, JSON.parse(JSON.stringify(props.items))),
       selectedValue: '',
     };
   }
 
+  addRenderer = (items: any , cloneItems: any) => {
+    items.forEach((it: any, ind: number) => {
```
Comment (dheeraja00) [inline src/components/ComboBox/ComboBox.tsx]: @ankitsaraogi use full form `index` & `item` or something else

File: src/components/Table/expression.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
+
+const getPartExpression = function (object: string, data: any) {
```
Comment (dheeraja00) [inline src/components/Table/expression.ts]: @ankitsaraogi can we use arrow function here?

Comment (ankitsaraogi) [reply src/components/Table/expression.ts]: @dheeraja00 : there is no need of arrow function as I dont want to pass this context inside function.

**Comment by ankitsaraogi** [comment]
@dheeraja00 : Please merge if no issue is seen.

**Pull request merged**